src/libs/ajax.js

```diff
@@ -989,12 +945,6 @@ export const Votes = {
     return res.json();
   },
 
-  getDarFinalAccessVote: async (requestId) => {
-    const url = `${await Config.getApiUrl()}/api/dataRequest/${requestId}/vote/final`;
```

Comment:
unused

Reply:
This means we can remove https://github.com/DataBiosphere/consent/blob/develop/src/main/java/org/broadinstitute/consent/http/resources/DataRequestVoteResource.java#L173